client-side/src/Components/studentComponent/Gatepasses/LocalFlexible/Form/LFform.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState, useReducer } from "react";
+import ReactDropdown from "react-dropdown";
+import moment from 'moment';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import "./style.scss";
+
+
+const LFform = (props) => {
+  const[hour,settime]=useState(new Date().getHours());// present time
+  const[minute,setminute]=useState(new Date().getMinutes());//present minute
+  const[max_time,setmaxTime]=useState([]);//maximum time accepted from backend
```

Comment:
CamelCase and "_" is missing from setState

Reply:
done